server/devdocs/index.js

```diff
@@ -1,20 +1,22 @@
 /**
  * Extenal dependencies
  */
-var express = require( 'express' ),
-	fs = require( 'fs' ),
-	fspath = require( 'path' ),
-	marked = require( 'marked' ),
-	lunr = require( 'lunr' ),
-	find = require( 'lodash/find' ),
-	escapeHTML = require( 'lodash/escape' );
+import express from 'express';
+import fs from 'fs';
+import fspath from 'path';
+import marked from 'marked';
+import lunr from 'lunr';
+import find from 'lodash/find';
+import escapeHTML from 'lodash/escape';
+import striptags from 'striptags';
 
 /**
  * Internal dependencies
  */
-var	config = require( 'config' ),
-	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
-	searchIndex = require( 'devdocs/search-index' ),
+import config from 'config';
+import searchIndex from 'devdocs/search-index';
+
+var	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
```

Comment:
>Checked this again, it is a single space, not sure why the diff shows it like this...

It displays as though it were a single space in my editor, but it's definitely a tab.